src/Razor/src/Microsoft.VisualStudio.Editor.Razor/Documents/EditorDocumentManagerListener.cs

```diff
@@ -187,28 +178,29 @@ await _dispatcher
                 {ex.StackTrace}
                 """);
         }
+
+        return Task.CompletedTask;
     }
 
     private void Document_ChangedInEditor(object sender, EventArgs e)
     {
-        _ = Document_ChangedInEditorAsync(sender, CancellationToken.None);
+        Document_ChangedInEditorAsync(sender, CancellationToken.None).Forget();
```

Comment:
Do we need to track any exceptions happening here?

Reply:
I'm not sure what I could do to make that happen here. Do you have ideas?

Reply:
Hmmm... I always forget that [FileAndForget](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.vstasklibraryhelper.fileandforget?view=visualstudiosdk-2022) is from the vs task library. We're still in the VS layer here though so maybe you can use it? It also seems like my usage is wrong when I did it for the snippets service. I'll get a PR to fix that up

Reply:
If I recall, FireAndForget() requires the VS ThreadHelper to be initialized. Am I remembering correctly?

Reply:
Filed #10109